src/components/ActivityCreation/ActivityCreationSideBar.tsx

```diff
@@ -59,7 +59,7 @@ const ActivityCreationSideBar = ({
           <LinkIcon className="mt-1 size-4 opacity-0 transition-opacity duration-300 group-hover:opacity-100" />
         </Link>
         <Link
-          className="group flex items-center gap-2 pl-6"
+          className="group flex w-fit items-center gap-2 pl-6"
           href="#activity-setting"
```

Comment:
```suggestion
          href="#activity"
```

Reply:
fix it : 6b854e58eccccd4c67c3c6f573ec6b97a0395010